### PR TITLE
Move custom domain post job to Github Actions

### PR DESCRIPTION
**Pull request by kolodziejczak**
**Description**

Changes proposed in this pull request:

- Copy and adjust script responsible for running job on Github Actions
- Add action and job for custom domain gcp test

### CODE REVIEW

File: tests/integration/scripts/custom-domain-gardener-gcp-gh.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+export DISK_SIZE=50
+export DISK_TYPE="pd-standard"
+export SCALER_MAX=3
+export SCALER_MIN=1
+export GARDENER_PROVIDER="gcp"
+export GARDENER_REGION="europe-west3"
+export GARDENER_ZONES="europe-west3-c,europe-west3-b,europe-west3-a"
+export GARDENER_PROVIDER_SECRET_NAME="goat"
+export GARDENER_PROJECT_NAME="goatz"
+export GARDENER_CLUSTER_VERSION="1.26.9"
```
Comment (barchw) [inline tests/integration/scripts/custom-domain-gardener-gcp-gh.sh:19]: Make configurable

Comment (barchw) [reply tests/integration/scripts/custom-domain-gardener-gcp-gh.sh:19]: Leaving it for follow up

File: tests/integration/scripts/provision-gardener-gh.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    MACHINE_TYPE
+    DISK_SIZE
+    DISK_TYPE
+    SCALER_MAX
+    SCALER_MIN
+)
+
+check_required_vars "${requiredVars[@]}"
+
+# Install Kyma CLI in latest version
```
[the remark this answers is not in the document]
Comment (barchw) [reply tests/integration/scripts/provision-gardener-gh.sh:39]: Leaving it for follow up

**Review by barchw — CHANGES_REQUESTED** [review]

**Review by barchw — DISMISSED** [review]

File: docs/contributor/04-30-ci-cd.md [hunk 3]
```diff
@@ -17,6 +17,13 @@ There are two environments configured:
 - 'restricted' - used when an outside collaborator runs a job in the repository. The run must be approved by
   @kyma-project/goat.
 
+## Prerequisites Github Actions secrets
```
Comment (nataliasitko) [inline docs/contributor/04-30-ci-cd.md:20]:
```suggestion
## Prerequisites for GitHub Actions Secrets
```

File: docs/contributor/04-30-ci-cd.md [hunk 4]
```diff
@@ -17,6 +17,13 @@ There are two environments configured:
...
+
+### Gardener specific
```
Comment (nataliasitko) [inline docs/contributor/04-30-ci-cd.md:22]:
```suggestion
### Gardener Specific
```

File: docs/contributor/04-30-ci-cd.md [hunk 5]
```diff
@@ -17,6 +17,13 @@ There are two environments configured:
...
+
+- `GARDENER_TOKEN` - Kubeconfig token used to provision test purpose shoots and later cleanup
```
Comment (nataliasitko) [inline docs/contributor/04-30-ci-cd.md:24]:
```suggestion
- `GARDENER_TOKEN` - Kubeconfig token used to provision test-purpose shoots and for later cleanup
```

File: docs/contributor/04-30-ci-cd.md [hunk 6]
```diff
@@ -17,6 +17,13 @@ There are two environments configured:
...
+- `DNS_SECRET_JSON` - DNS ServiceAccount used to set up custom domain through Gardener capabilities (BASE64 Encoded)
```
Comment (nataliasitko) [inline docs/contributor/04-30-ci-cd.md:25]:
```suggestion
- `DNS_SECRET_JSON` - DNS ServiceAccount used to set up the custom domain using Gardener capabilities (Base64 encoded)
```

**Pull request merged**